addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3608,7 +3608,7 @@ msgid "Singles"
 msgstr ""
 
 msgctxt "#1051"
-msgid "Enter web address"
+msgid "Enter Web address"
```

Comment:
well it´s not as one-sided as you try to make it here:
https://english.stackexchange.com/questions/120869/should-i-capitalize-the-word-web-in-this-sentence
seems as if capitalizing it is more some habit from the past.